functions/lib/blockiesSVG.js

```
@@ -92,7 +114,17 @@ function renderIdenticon(opts) {
             svg += '<rect x="' + col * opts.scale + '" y="' + row * opts.scale + '" width="' + opts.scale + '" height="' + opts.scale + '" fill="' + fill + '"/>';
         }
     }
-    return svg + '</svg>';
+
+    const iconBody = svg + '</svg>';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)